### PR TITLE
Markdown conversion for `pt-BR` - Replace - Accessibility section :warning:  Do not squash :warning: 

### PR DESCRIPTION
**Pull request by SphinxKnight**
Follow-up of https://github.com/mdn/translated-content/pull/8470

### CODE REVIEW

**Comment by SphinxKnight** [comment]
Please :warning: Do not squash merge :warning:

File: files/pt-br/web/accessibility/aria/roles/slider_role/index.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+title: Usando o slider role
+slug: Web/Accessibility/ARIA/Roles/slider_role
+translation_of: Web/Accessibility/ARIA/ARIA_Techniques/Using_the_slider_role
+original_slug: Web/Accessibility/ARIA/ARIA_Techniques/Using_the_slider_role
+---
+### Descrição
+
+Essa técnica demosntrará como usar o [slider](http://www.w3.org/TR/wai-aria/roles#slider) role.
+
+The` slider`` `role is used for markup that allows a user to select a value from within a given range. The slider role is assigned to the "thumb," the control that is adjusted to change the value. Typically, another element is styled to visually represent the range of possible valued, and the thumb positioned visually to represent the value within that range. As the user interacts with the thumb, the application must programmatically adjust the slider's `aria-valuenow` (and possible `aria-valuetext`) attribute to reflect the current value. See the [Examples](#examples) section below for more information.
```
Comment (josielrocha) [inline files/pt-br/web/accessibility/aria/roles/slider_role/index.md:11]: Hi @SphinxKnight,
Thank you for your contribution

Could you please fix the problemas below?

```suggestion
### Descrição

Essa técnica demonstrará como usar o [slider](http://www.w3.org/TR/wai-aria/roles#slider) role.

The `slider` role is used for markup that allows a user to select a value from within a given range. The slider role is assigned to the "thumb," the control that is adjusted to change the value. Typically, another element is styled to visually represent the range of possible valued, and the thumb positioned visually to represent the value within that range. As the user interacts with the thumb, the application must programmatically adjust the slider's `aria-valuenow` (and possible `aria-valuetext`) attribute to reflect the current value. See the [Examples](#examples) section below for more information.
```

File: files/pt-br/web/accessibility/aria/aria_techniques/index.md [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+- menubar
+- [radiogroup (veja a função radio)](/en/Accessibility/ARIA/ARIA_Techniques/Using_the_radio_role "en/ARIA/ARIA_Techniques/Using_the_radio_role")
+- [tablist](/pt-BR/docs/Web/Accessibility/ARIA/Roles/Tablist_Role) (incluindo as funções [tab](/pt-BR/docs/Web/Accessibility/ARIA/Roles/Tab_Role) and [tabpanel](/pt-BR/docs/Web/Accessibility/ARIA/Roles/Tabpanel_Role))
+- [tree](/pt-BR/docs/Web/Accessibility/ARIA/Roles/Tree_Role)
+- treegrid
+
+### Funções da estrutura de documento
+
+-
+- [application](/pt-BR/docs/Web/Accessibility/ARIA/Roles/Application_Role)
```
Comment (josielrocha) [inline files/pt-br/web/accessibility/aria/aria_techniques/index.md:57]: Removing empty list item

```suggestion
### Funções da estrutura de documento

- [application](/pt-BR/docs/Web/Accessibility/ARIA/Roles/Application_Role)
```

**Review by josielrocha — APPROVED** [review]
LGTM ;)

Thank you @SphinxKnight!

**Pull request merged**